core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/recipe/RecipeEngine.java

```diff
@@ -60,11 +60,15 @@ public void executePreClusterManagerRecipes(Stack stack, Collection<Recipe> reci
     public void executePostAmbariStartRecipes(Stack stack, Collection<Recipe> recipes) throws CloudbreakException {
         if ((stack.getCluster() != null && ldapConfigService.isLdapConfigExistsForEnvironment(stack.getEnvironmentCrn(), stack.getName()))
                 || recipesFound(recipes, POST_CLOUDERA_MANAGER_START)) {
+            // The ldapConfigService check forces this to run ALWAYS, which adds 10 seconds.
```

Comment:
There is a comment that we use this phase for LDAP sync, but I can't find it in the code so I guess it's not valid anymore.